lib/new-flatfile.ts

```diff
@@ -131,9 +160,9 @@ export const listSpaces = async ({
   }
 };
 
-export const getSpace = async (spaceId: string) => {
+export const getSpace = async (flowName: string, spaceId: string) => {
```

Comment:
If you have two parameters of the same type, I would change this into an object parameter, so that you don't accidentally swap the values and not know

Reply:
Makes sense. Edited.